paket.lock

```diff
@@ -21,8 +21,8 @@ NUGET
     Expecto.TestResults (8.13.2)
       Expecto (>= 8.13.2)
       FSharp.Core (>= 4.6)
-    FSharp.Compiler.Service (43.7.400)
-      FSharp.Core (7.0.400)
+    FSharp.Compiler.Service (43.8.300)
+      FSharp.Core (8.0.300)
```

Comment:
@TheAngryByrd two questions here:

* This is quite a bump in terms of allowed FSharp.Core versions. Library guidelines suggest using the lowest-available FSharp.Core to help compat, but F.C is tightly bound to FCS. Is this fine for the ecosystem of consumers we have?
* Since this FCS provides snapshots, do we want to grab/copy/modify the code you have in FSAC and provide helper functions here for mapping cracked projects to snapshots so that others can make use of it?

Reply:
> This is quite a bump in terms of allowed FSharp.Core versions. Library guidelines suggest using the lowest-available FSharp.Core to help compat, but F.C is tightly bound to FCS. Is this fine for the ecosystem of consumers we have?

Yeah this part always kind of sucks. 

The amount of surface area of FCS we use is pretty small and almost feels like it should be delivered as a source dependency. However if we want ProjectSnapshot we have to bump it and just deal with it unless we want to ship versions based on TFM or something. Speaking of ProjectSnapshot...

> Since this FCS provides snapshots, do we want to grab/copy/modify the code you have in FSAC and provide helper functions here for mapping cracked projects to snapshots so that others can make use of it?

While I don't necessarily have a problem with it there's a few things to consider:

1. FCS already provides a F# Options -> ProjectSnapshot which might be enough for most people's needs. It's another call but not really that detrimental. 
2. FSAC's implementation heavily tied to F# Data Adaptive. The Adaptive stuff is beneficial for longer lived services like FSAC, but for shorter one shot tools it's less necessary. And for longer lived services, I think that's what ProjectSystem is supposed to be.

Reply:
Excellent - I did a quick spike of the adaptive stuff and came to the same conclusions. Thanks for sanity-checking me.  I'll go ahead with this as it is then so we can continue the rollout.